.github/workflows/ci.yaml

```diff
@@ -35,6 +35,23 @@ jobs:
       - name: Test
         run: pnpm test
 
+  test-e2e:
+    name: Test E2E
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: ./.github/actions/setup-and-build
+
+      - name: Install Playwright Dependencies
+        run: pnpm exec playwright install chromium --with-deps
+        working-directory: e2e
+
+      - name: Test
+        run: pnpm test
+        working-directory: e2e
```

Comment:
I think we could remove those `working-directory` if we do:

```yaml
run: pnpm --filter=tutorialkit-e2e exec playwright install chromium --with-deps
```

and:

```yaml
run: pnpm test:e2e
````

Reply:
Sure I can apply this, but isn't `working-directory` a bit cleaner and simpler here?

Reply:
I find it slightly better because when you wonder what CI runs, you can just copy that command and execute it in your terminal without wondering in which folder you should be. It will work regardless of your current working directory (as long as it's a subfolder of the repo ofc).

So saying you always run `pnpm test:e2e` is shorter than saying "`pnpm test` but make sure to `cd ./e2e` ah wait, you need to `cd ../../e2e` because you're in `astro`"